src/Todo.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Todo({ todo, onClick }) {
```

Comment:
```suggestion
function Todo({ { text, id }, onClickDelete }) {
```

구조분해 할당과 명확한 onClick 함수명을 적용해봤습니다.

Reply:
의미론적으로 좀 더 고민해 보겠습니다.